R/readAndrew2019.R

```diff
@@ -0,0 +1,15 @@
+#' Read Data from Andrew's 2019 paper
+#' https://zenodo.org/records/11207133
+#' Last version update 17.05.2024
+#' Dataset update from:
+#' Andrew, R.M., 2019. Global CO2 emissions from cement production, 1928–2018.
+#' Earth System Science Data 11, 1675–1710. https://doi.org/10.5194/essd-11-1675-2019.
+#' @author Bennet Weiss
+readAndrew2019 <- function() {
+  path <- file.path("v1", "1. annual_cement_production.csv")
+  data <- readr::read_csv(path)
```

Comment:
can you set `  data <- suppressMessages(readr::read_csv(path))` to avoid console output spamming the inputdata log?